generated/nifake/_library.py

```diff
@@ -275,13 +275,13 @@ def niFake_Initiate(self, vi):  # noqa: N802
                 self.niFake_Initiate_cfunc.restype = ViStatus  # noqa: F405
         return self.niFake_Initiate_cfunc(vi)
 
-    def niFake_MultipleArrayTypes(self, vi, output_array_size, output_array, output_array_of_fixed_length, input_array_sizes, input_array_of_floats, input_array_of_integers):  # noqa: N802
+    def niFake_MultipleArrayTypes(self, vi, output_array_size, output_array, output_array_of_fixed_length, input_array_sizes, input_array_of_integers, input_array_of_floats):  # noqa: N802
```

Comment:
For testing, could you have 3 or 4, rather than 2 input arrays whose sizes must match?